CHANGELOG/7.5.md

```diff
@@ -0,0 +1,34 @@
+# 7.5 Changelog
+
+## [7.5.0] - 2025-01-21
+
+### Build and Packaging Improvements
+
+<details>
+
+<summary>
+
+<p>We thank the following contributors!</p>
```

Comment:
When no one to thank for here, change to call out the .NET SDK version used for this release. See existing releases as examples.